lib/data/queries/application/getMscApplicationByProgramByYear.ts

```diff
@@ -0,0 +1,25 @@
+import prisma from "../../../../prisma/client";
+import { Prisma } from "@prisma/client";
+
+const getMscApplicationsByProgramByYear = async (
+  program: string,
+  year: number,
+) => {
+  return prisma.application.groupBy({
+    by: ["examYear"],
+    _count: {
+      _all: true,
+    },
+    where: {
```

Comment:
I think we should filter by `status` as well. We only want to consider applicants who finished the course.